ai-gateway/app/services/llm_generator.py

```
@@ -0,0 +1,46 @@
+
+    try:
+        async with httpx.AsyncClient(timeout=120.0) as client:
+            response = await client.post(f"{OLLAMA_API_URL}/api/generate", json=payload)
+            response.raise_for_status()
+            
+            data = response.json()
+            # The LLM returns a JSON string in the 'response' field because we used format="json"
+            import json
+            return json.loads(data["response"])
+            
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)